tests/backtest/test_portfolio.py

```diff
@@ -0,0 +1,37 @@
+"""Test the Portfolio object."""
+
+from coin_test.backtest import Portfolio
+
+assets = {"BTC": 2, "ETH": 1.5, "DOGE": 358.25}
+cash = 10000.0
```

Comment:
Globals are bad practice, lets use [fixtures ](https://docs.pytest.org/en/6.2.x/fixture.html)instead. Ping me if you want some examples for this.